src/components/vamcFacility/template.test.tsx

```diff
@@ -177,44 +177,10 @@ describe('HealthCareLocalFacility with valid data', () => {
     )
     expect(jsonLdScript).toBeInTheDocument()
 
-    // Parse the JSON to verify content
-    const jsonData = JSON.parse(jsonLdScript?.innerHTML || '{}')
-
-    // Test essential properties
-    expect(jsonData['@context']).toBe('https://schema.org')
-    expect(jsonData['@type']).toBe('Place')
-    expect(jsonData.name).toBe(mockData.title)
-    expect(jsonData.telephone).toBe(mockData.mainPhoneString)
-
-    // Test address
-    expect(jsonData.address['@type']).toBe('PostalAddress')
-    expect(jsonData.address.streetAddress).toContain(
-      mockData.address.address_line1
-    )
-    expect(jsonData.address.addressLocality).toBe(mockData.address.locality)
-    expect(jsonData.address.addressRegion).toBe(
-      mockData.address.administrative_area
-    )
-    expect(jsonData.address.postalCode).toBe(mockData.address.postal_code)
-
-    // Test geo coordinates
-    expect(jsonData.geo['@type']).toBe('GeoCoordinates')
-    expect(jsonData.geo.latitude).toBe(mockData.geoLocation?.lat)
-    expect(jsonData.geo.longitude).toBe(mockData.geoLocation?.lon)
-
-    // Test facility ID
-    expect(jsonData.branchCode).toBe(mockData.facilityLocatorApiId)
-
-    // Test opening hours
-    expect(Array.isArray(jsonData.openingHoursSpecification)).toBe(true)
-    expect(jsonData.openingHoursSpecification.length).toBe(
-      mockData.officeHours.length
-    )
-
-    // Test first opening hours entry
-    const firstHours = jsonData.openingHoursSpecification[0]
-    expect(firstHours['@type']).toBe('OpeningHoursSpecification')
-    expect(firstHours.dayOfWeek).toContain('https://schema.org/')
+    // Verify it contains valid JSON
+    expect(() => {
+      JSON.parse(jsonLdScript?.innerHTML || '{}')
+    }).not.toThrow()
```

Comment:
Now that we're not checking the contents of the script in this test, we should make sure that it's _not_ just `{}`.

Reply:
https://github.com/department-of-veterans-affairs/next-build/pull/1490/commits/6cd065e0bd401837da175b886eef3e1c3e281996